### PR TITLE
Feature/taxonomies sidebar

### PR DESCRIPTION
**Pull request by triptec**
Changed parentId to string because model ids are string.

### CODE REVIEW

File: packages/storefront/addon/components/spree-taxonomy.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+export default Ember.Component.extend({
+  layout: layout,
+  childTaxons: function(){
+    var taxons = this.get('taxonomy.taxons');
+    var parentTaxon = taxons.findBy('parentId', null);
+    var childTaxons = taxons.filter(function(taxon){
+      return taxon.get('parentId') === parentTaxon.get('id');
+    });
+    return childTaxons;
+  }.property('taxonomy.taxons')
```
Comment (hhff) [inline packages/storefront/addon/components/spree-taxonomy.js]: Can we please change this to the non-prototype syntax?

```
childTaxons: Ember.computed('taxonomy.taxons', function() {...
```

More info: http://reefpoints.dockyard.com/2015/03/22/tips-for-writing-ember-addons.html

File: packages/storefront/addon/templates/components/spree-taxonomies.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<nav id="taxonomies" class="sidebar-item">
+  <ul class="side-nav">
+    {{#each taxonomy in taxonomies}}
+        {{spree-taxonomy taxonomy=taxonomy}}
```
Comment (hhff) [inline packages/storefront/addon/templates/components/spree-taxonomies.hbs]: stylistic nitpick, but this guy is too far indented

File: packages/storefront/app/templates/spree/taxonomies.hbs [hunk 3]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (hhff) [inline packages/storefront/app/templates/spree/taxonomies.hbs]: probably don't need this template for now

File: packages/core/app/models/taxon.js [hunk 4]
```diff
@@ -4,6 +4,6 @@ export default DS.Model.extend({
   name: DS.attr('string'),
   prettyName: DS.attr('string'),
   permalink: DS.attr('string'),
-  parentId: DS.attr('number'),
+  parentId: DS.attr('string'),
```
Comment (hhff) [inline packages/core/app/models/taxon.js]: :+1:

**Comment by hhff** [comment]
What's your status on this @triptec - all looks good to me?  Is there still work to be done here?

**Comment by triptec** [comment]
There's still things to be done but I think it's a good start and should be pulled.

**Comment by hhff** [comment]
Can do!  Can you please squash commits, and close tag #53 with the commit message?

**Pull request merged**